example/kotlinlib/module/8-kotlin-compiler-plugins/build.mill

```diff
@@ -15,10 +15,12 @@ object foo extends KotlinModule {
     ivy"org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.3"
   )
 
+  def kotlinCompilerEmbeddable = true
```

Comment:
Do you enable it to show-case it, or does is actually fix/improves anything?

Reply:
just to show case it, there are 2 such examples, I've enabled it only here. I can skip this and do it in the dagger example of the following PR

Reply:
If it's not really needed, we should not use it, esp. since we plan to show it in the dagger example. In case we want to show it in isolation, there should be at least some docs pointing out the fact.

Reply:
reverted!